1-raw-data/mailinglists/getMailData.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+//TODO
+// Create tests
+// check the most recent file stored and pull only what isn't there
+// Run this monthly at start of new month to pull all new data
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"flag"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"google.golang.org/api/iterator"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+)
+
+type GCS struct {
```

Comment:
Does this struct need to be exported? If the struct name starts with a capital letter, it's exported and accessible beyond the package it's in.

More concrete info: https://golangbyexample.com/exported-unexported-fields-struct-go/

Reply:
Good to know. Keeping as is for now because potentially but noted on the mechanics.